packages/story-editor/src/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -222,7 +228,13 @@ function TextSetsPane({ paneRef }) {
         />
       </FullWidthWrapper>
       <TextSetsWrapper>
-        <TextSets paneRef={paneRef} filteredTextSets={filteredTextSets} />
+        {!filteredTextSets?.length && !areTextSetsLoading ? (
+          <EmptyContentMessage>
+            {__('No text sets available.', 'web-stories')}
```

Comment:
```suggestion
            {__('No Text Sets available.', 'web-stories')}
```

I _think_ this should be capitalized. 🤔